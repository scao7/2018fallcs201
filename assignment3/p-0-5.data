16 11 272 ; 20 29 411 ; 2 21 445 ; 11 11 341 ; 14 21 253 ; 
18 21 431 ; 0 5 417 ; 6 16 184 ; 0 10 286 ; 23 27 223 ; 
2 15 0 ; 31 10 472 ; 20 23 296 ; 31 16 158 ; 21 9 138 ; 
10 5 176 ; 15 18 330 ; 0 27 98 ; 10 1 48 ; 5 1 293 ; 
20 5 428 ; 31 26 202 ; 22 2 281 ; 7 13 197 ; 16 3 109 ; 
21 7 482 ; 8 29 420 ; 3 20 443 ; 4 16 151 ; 5 13 249 ; 
11 1 38 ; 5 23 291 ; 7 12 122 ; 20 22 124 ; 23 31 491 ; 
30 29 180 ; 28 5 417 ; 16 28 325 ; 0 20 88 ; 14 21 345 ; 
15 28 466 ; 7 23 157 ; 19 17 38 ; 9 1 406 ; 8 25 122 ; 
5 1 474 ; 10 30 340 ; 7 4 200 ; 27 12 208 ; 16 25 209 ; 
12 12 199 ; 3 9 106 ; 20 12 485 ; 22 22 265 ; 15 20 346 ; 
16 10 191 ; 14 31 428 ; 18 15 35 ; 31 16 102 ; 24 17 281 ; 
4 0 321 ; 14 30 466 ; 26 4 301 ; 16 29 6 ; 4 31 0 ; 
15 10 479 ; 14 27 123 ; 29 22 349 ; 13 24 188 ; 30 21 375 ; 
30 2 459 ; 29 28 408 ; 1 26 378 ; 30 12 260 ; 29 20 460 ; 
8 11 100 ; 3 18 7 ; 25 7 169 ; 17 15 491 ; 6 22 180 ; 
8 22 447 ; 5 26 341 ; 31 12 470 ; 11 24 23 ; 31 20 413 ; 
10 20 191 ; 28 4 343 ; 3 1 206 ; 19 28 111 ; 9 5 160 ; 
31 12 338 ; 25 17 114 ; 5 20 307 ; 29 23 70 ; 17 25 57 ; 
6 24 120 ; 10 31 90 ; 11 9 90 ; 8 12 322 ; 13 0 473 ; 
25 26 270 ; 10 28 26 ; 30 4 295 ; 22 10 161 ; 15 28 382 ; 
7 12 272 ; 6 7 430 ; 15 29 364 ; 27 23 95 ; 27 4 467 ; 
21 27 160 ; 17 17 473 ; 21 0 44 ; 0 21 398 ; 28 8 208 ; 
8 12 239 ; 15 22 182 ; 13 10 380 ; 4 9 243 ; 9 17 178 ; 
4 5 401 ; 21 19 398 ; 21 11 136 ; 11 1 125 ; 19 14 85 ; 
31 9 73 ; 21 12 414 ; 31 20 345 ; 8 12 14 ; 25 30 39 ; 
31 12 448 ; 18 14 12 ; 29 2 257 ; 30 12 182 ; 12 21 301 ; 
22 27 496 ; 2 9 39 ; 22 6 212 ; 2 17 231 ; 1 0 192 ; 
13 24 413 ; 27 20 260 ; 30 5 2 ; 10 11 239 ; 31 29 18 ; 
26 1 250 ; 
