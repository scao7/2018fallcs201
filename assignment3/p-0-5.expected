0: 16
1: 29(16)6
2: 23(29)70 31(29)18
3: 4(31)0 7(23)157 9(31)73 26(31)202
4: 2(9)39 3(9)106 11(9)90 12(7)122
5: 1(11)38 8(11)100 15(2)0 18(3)7 24(11)23
6: 6(24)120 10(1)48 13(24)188 14(18)12
7: 19(14)85 27(14)123 28(10)26
8: 0(27)98 17(19)38
9: 20(0)88 21(0)44 25(17)114
10: 22(20)124 30(25)39
11: 5(30)2
weight: 2200
